code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -920,6 +919,14 @@ GLOBAL_LIST_INIT(strippable_parrot_items, create_strippable_list(list(
 	ears = new /obj/item/radio/headset/headset_eng(src)
 	if(SStts.tts_enabled)
 		voice = pick(SStts.available_speakers)
+		if(SStts.pitch_enabled)
+			if(findtext(voice, "Woman"))
```

Comment:
this is not a viable long term solution, you'll want to leave a comment about this that it should be replaced by a tagging system whenever available.
For now, I'll let this pass, but I'm going to outright reject any future PRs that try to distinguish between voices by how they're named.

Reply:
oh yeah 100%